### PR TITLE
[IMP] add a README

### PR DESCRIPTION
**Pull request by clonedagain**
No big deal, still nice to have.

### CODE REVIEW

File: l10n_fr_siret/README.rst [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+numbers from the official SIRENE registry in France: SIRET, SIREN and NIC.
+These numbers identify each company and their subsidiaries, and are
+often required for administrative tasks.
+
+Usage
+=====
+
+On the Partner form, users will be able to enter the SIREN
+and NIC numbers, and the SIRET number will be calculated
+automatically.  The last digits of the SIREN and NIC are control keys:
```
Comment (rvalyi) [inline l10n_fr_siret/README.rst]: double space before The (ça blague plus à l'OCA hein)

**Comment by rvalyi** [comment]
:+1:  just double space to fix.

**Comment by pedrobaeza** [comment]
:+1:

**Comment by clonedagain** [comment]
Inserted paragraph break instead of double space to pretend I'm smart.

**Comment by yvaucher** [comment]
:+1:

**Pull request merged**